v1/pipeline/stages/cd/deployment-stage-config.yaml

```diff
@@ -1,36 +1,30 @@
+$schema: http://json-schema.org/draft-07/schema#
 title: DeploymentStageConfig
 type: object
 required:
-- execution
+- steps
 properties:
-  customDeploymentRef:
+  custom_deployment:
```

Comment:
should we call it deployment_template instead ?

Reply:
Sure. I will make the change.